migrations/1755100736.sh

```diff
@@ -0,0 +1,5 @@
+echo "Install omarchy-setup-cloudflare-dns script for DNS configuration"
+
+# Install DNS configuration script to /usr/local/bin for system-wide access
+sudo cp $HOME/.local/share/omarchy/bin/omarchy-setup-cloudflare-dns /usr/local/bin/
+sudo chmod +x /usr/local/bin/omarchy-setup-cloudflare-dns
```

Comment:
Won't need this either, but we should have a migration that sets the automatic setup for cloudflare back to nothing.